config/theme.go

```diff
@@ -84,5 +84,45 @@ func GetColor(key string) tcell.Color {
 func GetColorString(key string) string {
 	themeMu.RLock()
 	defer themeMu.RUnlock()
-	return fmt.Sprintf("#%06x", theme[key].TrueColor().Hex())
+	color := theme[key].TrueColor()
+	if color == tcell.ColorDefault {
+		return "-"
+	}
+	return fmt.Sprintf("#%06x", color.Hex())
+}
+
+// GetReadableColor Returns ColorBlack if the color is brighter than gray
+// otherwise returns ColorWhite
+func GetReadableColor(bg tcell.Color) tcell.Color {
+	if bg == tcell.ColorDefault {
+		return tcell.ColorWhite
+	}
+	r, g, b := bg.RGB()
+	luminance := 77*r + 150*g + 29*b + (1<<7)>>8
```

Comment:
```suggestion
	luminance := (77*r + 150*g + 29*b + 1<<7) >> 8
```
Gave you the wrong math before, sorry.